junebug/channel.py

```diff
@@ -142,3 +152,15 @@ def status(self):
         # TODO: Implement channel status
         status['status'] = {}
         return status
+
+    @inlineCallbacks
+    def send_message(self, amq_client, to_addr, content, **kw):
+        message = TransportUserMessage.send(
+            to_addr=to_addr, content=content, transport_name=self.id)
+        queue = '%s.outbound' % self.id
+        msg = yield amq_client.publish_message(message, routing_key=queue)
+        ret = {}
+        for key, val in msg.payload.iteritems():
+            if key in allowed_message_fields:
+                ret[key] = val
```

Comment:
This has to do with what we return, not what gets sent. At the moment, only `to_addr` and `content` are sent through. I didn't want to make this PR too big with validation things, but we'll need to make sure to pass all the fields given to us in the message.